python/pyarrow/lib.pyx

```diff
@@ -169,6 +169,9 @@ include "builder.pxi"
 # Column, Table, Record Batch
 include "table.pxi"
 
+# Compute registries
+include "compute.pxi"
```

Comment:
For the record, why is this addition necessary? Can Substrait instead directly import these declarations?